blog/_posts/2020-09-09-github-issues.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Issues? GitHub!"
+layout: archive
+author: Axel Naumann
+---
+
+First of all: a big THANK YOU for submitting bugs for ROOT.
+We appreciate the effort: finding back the issue submission web page, explaining what the problem is, and sometimes even providing a reproducer.
+
+To make it easier for you to submit issues, we now switched from ~~Jira~~ to [GitHub Issues](https://github.com/root-project/root/issues/new).
+We hope that having a GitHub account is so common these days that you're already logged in and used to GitHub's interfaces.
+
+O, and Markdown instead of `{code}`. Phew!
+
+Starting today, new bugs will only be reported in GitHub.
+We will keep fixing the existing issues in Jira until they are all resolved.
+Jira now displays a message, asking you to instead use GitHub for issue submission.
+
+For us, that change will help in triaging issues, tracking progress, and most importantly bringing issue handling way closer to our code.
+Pull requests, commits, issues, code: they can all cross-reference now, giving a consistent view.
+
+Moving to a closed-source universe like GitHub isn't exactly obvious for us.
+(But then again we're coming from Jira...)
+On the other hand we did not want to use CERN's GitLab because that requires a CERN account for issue submission.
```

Comment:
```suggestion
On the other hand we did not want to use CERN's GitLab because that requires a CERN account for issue submission, and exactly that was one of the biggest problems with Jira.
```